src/routes/notification/types/index.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import NotificationTypeController from '../../../controllers/NotificationTypeController';
+
+const router = express.Router();
+
+router.get('/', NotificationTypeController.getTypes);
+router.get('/:id', NotificationTypeController.getTypes);
+router.post('/', NotificationTypeController.createType);
+router.put('/:id', NotificationTypeController.updateType);
+router.delete('/:id', NotificationTypeController.deleteType);
+
+export default router;
```

Comment:
These routes need to be registered in the src/makeroutes.js file otherwise they are not accessible.

Reply:
I checked it in postman, these routes work fine. I think separate file looks good but if you want I can't move it to makeroutes.js